leshan-core/src/main/java/org/eclipse/leshan/core/node/codec/text/LwM2mNodeTextDecoder.java

```diff
@@ -114,4 +119,4 @@ private static Object parseTextValue(String value, Type type, LwM2mPath path) th
             throw new CodecException("Could not handle %s value with TEXT encoder for resource %s", type, path);
         }
     }
-}
+}
```

Comment:
end-of-line lost?

Reply:
Not really intended so I will re-add it.